### ISSUE TITLE
So much "stream handler: don't want incoming peer",is that normal?

### ISSUE THREAD

**Issue by xiaoyell** [ethersphere/bee#1916]
#### Summary
It's so much logs like "stream handler: don't want incoming peer 6e43fb95b314a37531b6a656dae31e35c7be35a69c0c3f6eff81f013d2b4f66c. disconnecting" when I running a node.
It that normal or not?
And logs shown some ERROR like " stream handler: handshake: unable to handshake with peer id 16Uiu2HAmQ47RGNfBQF2W1w6SuZH2LJNJH514TN1DsTfu9wHAV7gF"

#### Steps to reproduce
I start it by running"bee start --config  /etc/bee/bee.yaml" in a screen command.

#### Actual behavior

Here's the yaml file:
clef-signer-enable: true
clef-signer-endpoint: /var/lib/bee-clef/clef.ipc
config: /etc/bee/bee.yaml
data-dir: /var/lib/bee
debug-api-addr: 127.0.0.1:1635
debug-api-enable: true
full-node: true
p2p-addr: :1634
p2p-ws-enable: false
password-file: /var/lib/bee/password
swap-enable: true
swap-endpoint: https://goerli.infura.io/v3/XXXXXXXXX
verbosity: info
welcome-message: "It's Four's 2nd bee...BZZ.BZZ"

**Comment by kennylbj** [comment]
Same to me

**Comment by acud** [comment]
This is fine. Your nodes will accept only a limited number of peers on bins which are shallower than your kademlia depth. Or in plain english: everything is in order :+1:
